### ISSUE TITLE
Command `show-trace` ends with "Trace file does not exist!" error if path is not absolute

### ISSUE THREAD

**Issue by JaPyR** [MarketSquare/robotframework-browser#2170]
**Describe the bug**
Recently I've update Browser library to 13.2 and noticed that `show-trace` command ends with error in situations that worked previously.

```robotframework
# BugInTracing.robot
*** Settings ***
Library    Browser

*** Test Cases ***
BugInTracing
    ${browser_args}    Create Dictionary
    ...    headless=${True}
    New Browser    headless=${True}
    New Context    tracing=${CURDIR}${/}traces.zip
    New Page    https://ddg.gg/
```

**Screenshots**
![image](https://user-images.githubusercontent.com/7773301/177955468-d9deda09-a3a0-4e7a-b8a1-1a4a2d334884.png)

**Desktop (please complete the following information):**
 - OS: Windows 11

**Comment by aaltat** [comment]
This is true and bit annoying. It could be enhanced and hopefully is not that difficult. 

@allcontributors please add @JaPyR for bugs.

**Comment by JaPyR** [comment]
Wouldn't be enough `Path(file).absolute().as_posix()` ?
https://github.com/MarketSquare/robotframework-browser/blob/7017e8181fa8c5014cbb6a482929e452da384f9e/Browser/entry.py#L175-L187

**Comment by aaltat** [comment]
Most likely that should work.
